examples/minimal-mdns/advertiser.cpp

```diff
@@ -33,19 +33,28 @@ enum class AdvertisingMode
 {
     kCommisioning,
     kOperational,
+#if CHIP_ENABLE_COMMISIONABLE_DISCOVERY
```

Comment:
No need for an ifdef in example code. Please remove - this conditional occurs in too many places of the code.